Scripts/Services/XmlSpawner/XmlSpawner2.cs

```diff
@@ -11370,9 +11370,11 @@ private class SpawnerTimer : Timer
             public SpawnerTimer(XmlSpawner spawner, TimeSpan delay)
                 : base(delay)
             {
-
-                // reduce timer priority if spawner is inactivated
-                if (spawner.IsInactivated)
+                if (spawner.IsInactivated) // reduce timer priority if spawner is inactivated
+                {
+                    Priority = TimerPriority.FiveSeconds;
+                }
+                else if (spawner.CurrentCount == spawner.MaxCount)
```

Comment:
Added spawner.CurrentCount == spawner.MaxCount

Regardless of what your re-spawn time is the DEFAULT built in check is 1 second. 

Priority = spawner.BasePriority; = 1 second